Gemfile

```diff
@@ -8,8 +8,11 @@ ruby '3.0.2'
 gem 'rails', '~> 6.1.5'
 
 gem 'bootsnap', '>= 1.4.4', require: false # https://rubygems.org/gems/bootsnap
+gem 'bcrypt'
+gem 'discard', '~> 1.2'
 gem 'dotenv-rails', '~> 2.7', '>= 2.7.6' # https://rubygems.org/gems/dotenv-rails
 gem 'jsonapi-serializer', '~> 2.2' # https://rubygems.org/gems/jsonapi-serializer
+gem 'jwt'
```

Comment:
add comment # repo link

Reply:
And don't forget to add the gem version.